plugin/al/translators/CommonTranslatorOptions.h

```diff
@@ -22,7 +22,13 @@ static constexpr const char* const kMeshUvOnly = "Mesh UV Only"; ///< export mes
 static constexpr const char* const kMeshPointsAsPref
     = "Mesh Points as PRef"; ///< export mesh Points as PRef, duplicating "P"
 static constexpr const char* const kMeshColours = "Mesh Colours"; ///< export mesh Colour Sets
-static constexpr const char* const kMeshHoles = "Mesh Holes";     ///< export mesh face holes
+static constexpr const char* const kMeshDefaultColourRGB = "Default RGB"; ///< default rgb values
+static constexpr const char* const kMeshDefaultColourA = "Default Alpha"; ///< default alpha values
+static constexpr const char* const kCustomColourThreshold
+    = "Custom Colour Threshold"; ///< default alpha values
```

Comment:
Comment here and on the next does not corresponds.

Reply:
Fixed.